### PR TITLE
LPS-161905 | master

### PR DESCRIPTION
**Pull request by Italo-gabriel**
https://issues.liferay.com/browse/LPS-161905

### CODE REVIEW

**Comment by Italo-gabriel** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testFunctional/macros/ObjectAdmin.macro [hunk 1]
```diff
@@ -37,6 +37,18 @@ definition {
 			locator1 = "Button#TAB_SUBMIT_BTN");
 	}
 
+	macro addLayoutTabViaUI {
```
Comment (evanilson-santana) [inline modules/apps/object/object-test/src/testFunctional/macros/ObjectAdmin.macro:40]: Hey, @Italo-gabriel this macro already exists, can you search by:  _ObjectAdmin.addTabFieldsOnLayout_

File: modules/apps/object/object-test/src/testFunctional/macros/ObjectAdmin.macro [hunk 2]
```diff
@@ -1345,6 +1357,18 @@ definition {
...
+		Click(locator1 = "ObjectAdmin#ADD_FIELD_ON_BLOCK");
+
+		Click(locator1 = "ObjectAdmin#LAYOUT_ENTRY_CHOOSE_OPTION");
+
+		for (var fieldsTypes : list "${fieldsTypesList}") {
+			AssertElementPresent(
+				key_optionName = "${fieldsTypes}",
+				locator1 = "ObjectAdmin#LAYOUT_ADD_FIELD_OPTION");
+		}
+	}
```
Comment (evanilson-santana) [inline modules/apps/object/object-test/src/testFunctional/macros/ObjectAdmin.macro:1370]: Can you make this assertion at the test level? This macro is specific for this scenario.

File: modules/apps/object/object-test/src/testFunctional/macros/ObjectPortlet.macro [hunk 3]
```diff
@@ -359,6 +359,10 @@ definition {
 			locator1 = "ObjectPortlet#SELECT_OBJECT");
 	}
 
+	macro selectObjectLayout {
+		ObjectPortlet.viewEntryDetails(entry = "${entry}");
+	}
```
Comment (evanilson-santana) [inline modules/apps/object/object-test/src/testFunctional/macros/ObjectPortlet.macro:364]: The same thing here. You don't need to create this macro. There's a macro that makes this. You can search by _ObjectAdmin.assertObjectLayout_.

File: modules/apps/object/object-test/src/testFunctional/paths/ObjectAdmin.path [hunk 4]
```diff
@@ -20,6 +20,11 @@
 		<td>//button[contains(@class,'btn btn-sm btn-secondary')]</td>
 		<td></td>
 	</tr>
+	<tr>
+		<td>ADD_TAB_BUTTON</td>
+		<td>//button[contains(@class,'btn-secondary') and contains(.,'Add Tab')]</td>
+		<td></td>
+	</tr>
```
Comment (evanilson-santana) [inline modules/apps/object/object-test/src/testFunctional/paths/ObjectAdmin.path:27]: Do you use this path in some steps of the test? You can use something like: `LexiconEntry.gotoAddNoSelectFrame();` to add tab on the layout

**Comment by epark18** [comment]
Hey @Italo-gabriel, are there any updates on this PR?

**Comment by Italo-gabriel** [comment]
> Hey @Italo-gabriel, are there any updates on this PR?
Hi, @epack18 I'm working on the changes that @evanilson asked me to make, but other demands have arisen and this activity has been a bit stopped for these days, I'll probably return to this activity on Thursday because I'm on PTO until tomorrow.

File: modules/apps/object/object-test/src/testFunctional/macros/ObjectAdmin.macro (hunk 1 above)
Comment (Italo-gabriel) [reply modules/apps/object/object-test/src/testFunctional/macros/ObjectAdmin.macro]: Done !

File: modules/apps/object/object-test/src/testFunctional/macros/ObjectAdmin.macro (hunk 2 above)
Comment (Italo-gabriel) [reply modules/apps/object/object-test/src/testFunctional/macros/ObjectAdmin.macro]: Done !

File: modules/apps/object/object-test/src/testFunctional/macros/ObjectPortlet.macro (hunk 3 above)
Comment (Italo-gabriel) [reply modules/apps/object/object-test/src/testFunctional/macros/ObjectPortlet.macro]: Done !

File: modules/apps/object/object-test/src/testFunctional/paths/ObjectAdmin.path (hunk 4 above)
Comment (Italo-gabriel) [reply modules/apps/object/object-test/src/testFunctional/paths/ObjectAdmin.path]: Done !

**Comment by Italo-gabriel** [comment]
ci:test:sf

**Comment by epark18** [comment]
ci:forward

**Pull request closed without merging**